pdc/apps/release/serializers.py

```diff
@@ -64,7 +64,7 @@ class ProductSerializer(StrictSerializerMixin, serializers.ModelSerializer):
 
     class Meta:
         model = Product
-        fields = ('name', 'short', 'active', 'product_versions', 'allowed_push_targets')
+        fields = ('name', 'short', 'active', 'product_versions', 'allowed_push_targets', 'id')
```

Comment:
Would it be OK to use name `pk` instead of `id`? Endpoint "release-variants" has `id` field which is string.